gooddata-sdk/gooddata_sdk/compute/model/execution.py

```diff
@@ -121,17 +187,19 @@ def __repr__(self) -> str:
         return f"ExecutionResult(paging={self.paging})"
 
 
-class ExecutionResponse:
+class BareExecutionResponse:
+    """
+    Holds ExecutionResponse from triggered report computation and allows reading report's results.
+    """
+
     def __init__(
         self,
         actions_api: apis.ActionsApi,
         workspace_id: str,
-        exec_def: ExecutionDefinition,
         response: models.AfmExecutionResponse,
     ):
         self._actions_api = actions_api
         self._workspace_id = workspace_id
-        self._exec_def = exec_def
 
         self._r: models.ExecutionResponse = response["execution_response"]
```

Comment:
Could we give it a more appropriate name rather than self._r, please?

Reply:
fixed in last commit